### ISSUE TITLE
dropdown cell will always refocus when scrolling grid 

### ISSUE THREAD

**Issue by chenghsuan** [angular-ui/ng-grid#2538]
Dear all,

I found dropdownEditor always refocus when scrolling grid (`ui-grid v3.0.0-RC.18`) and try to solve this problem . [Plunker](http://plnkr.co/edit/Zifa33qv1NTxoNwGbGDQ?p=preview)

I notice that `uiGridRenderContainer directive` will auto re-focus cell. If I comment out below line will work normally.
```
              $scope.$on(uiGridConstants.events.GRID_SCROLL, function (evt, args) {
                // Skip if there's no currently-focused cell
                if (uiGridCtrl.grid.api.cellNav.getFocusedCell() == null) {
                  return;
                }
                // We have to wrap in TWO timeouts so that we run AFTER the scroll event is resolved.
                $timeout(function () {
                  $timeout(function () {
                    // Get the last row+col combo
                    var lastRowCol = uiGridCtrl.grid.api.cellNav.getFocusedCell();

                    // If the body element becomes active, re-focus on the render container so we can capture cellNav events again.
                    //   NOTE: this happens when we navigate LET from the left-most cell (RIGHT from the right-most) and have to re-render a new
                    //   set of cells. The cell element we are navigating to doesn't exist and focus gets lost. This will re-capture it, imperfectly...
                    if ($document.activeElement === $document.body) {
                      $elm[0].focus();
                    }

                    // Re-broadcast a cellNav event so we re-focus the right cell
                     uiGridCtrl.cellNav.broadcastCellNav(lastRowCol); -------->this line
                  });
                });
              });
            }
          };
        }
      };
    }]);
```

**Comment by PaulL1** [comment]
I think this is a duplicate of #2423, and also #2467 (ignoring the starting discussion on that issue about scrolling).

**Comment by eldhonest** [comment]
Please let us know do we have a fix for this issue, I am using version 4.4.4 and things messed up with the scrolling. The edited cell will always refocus when scrolling grid horizontally.  I tried to fix the issue by commenting "uiGridCtrl.cellNav.broadcastCellNav(lastRowCol);" but it lead to tab/edit issue
